pkg/cmd/server/api/types.go

```diff
@@ -673,6 +673,17 @@ type GitHubIdentityProvider struct {
 	ClientSecret string
 }
 
+type GitLabIdentityProvider struct {
+	unversioned.TypeMeta
+
```

Comment:
Totally missed this one, sorry about that :frowning:

I added the field now to `GetMasterFileReferences`.